umm-spec-lib/src/cmr/umm_spec/acl_matchers.clj

```diff
@@ -127,6 +130,30 @@
          (or (nil? temporal)
              (matches-temporal-filter? :collection (u/get-real-or-lazy coll :TemporalExtents) temporal)))))
 
+(comment
```

Comment:
Did you mean to leave the comment in here?  
It looks like debugging code.

Reply:
If so can you delete line 156 to just allow 1 space and then put line 154 ) at the end of line 152.  Then delete line 151.

Reply:
I did mean to leave it in, but the question I have is do you find it interesting to see an example map?